### PR TITLE
Update base image rebased

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,9 +2,9 @@
 # CLA_BACKEND Dockerfile all environments
 #
 # Pull base image.
-FROM phusion/baseimage:0.9.11
+FROM phusion/baseimage:0.9.22
 
-MAINTAINER Peter Idah <peter.idah@digital.justice.gov.uk>
+MAINTAINER Platforms <platforms@digital.justice.gov.uk>
```
Comment (aliuk2012) [inline Dockerfile]: I think we can drop this line, I did for the cla_public

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Pull request merged**